sirepo/simulation_db.py

```diff
@@ -855,6 +855,8 @@ def user_dir_name(uid=None):
         return d
     return d.join(uid)
 
+def validate_name(data):
+    _validate_name(data)
```

Comment:
inline _validate_name(). I'm not sure why there's a need for a separate function.

Reply:
I had been calling this from server.py and forgot to delete once that went away